### PR TITLE
Fix asciidoc targets in RBE environment

### PR DESCRIPTION
**Pull request by schroederc**

### CODE REVIEW

File: kythe/docs/asciidoc.bzl [hunk 1]
```diff
@@ -36,7 +32,6 @@ def _asciidoc_impl(ctx):
                   ([ctx.file.example_script] if ctx.file.example_script else []) +
                   ctx.files.data),
         outputs = [ctx.outputs.out, logfile],
-        env = {"PATH": tool_path},  # so we can locate the binaries asciidoc needs
```
Comment (schroederc) [inline kythe/docs/asciidoc.bzl]: This leads the action to not being able to find standard tools like `mkdir`.

Comment (shahms) [reply kythe/docs/asciidoc.bzl]: This change will break macOS (which is why this was added in the first place).

Comment (schroederc) [reply kythe/docs/asciidoc.bzl]: Does that work for macOS?

Comment (shahms) [reply kythe/docs/asciidoc.bzl]: We'll see; I can take a look this afternoon to make sure.

Comment (shahms) [reply kythe/docs/asciidoc.bzl]: This works.

**Pull request merged**